src/gui/categoryfiltermodel.cpp

```diff
@@ -181,6 +181,12 @@ CategoryFilterModel::CategoryFilterModel(QObject *parent)
     connect(session, &Session::categoryRemoved, this, &CategoryFilterModel::categoryRemoved);
     connect(session, &Session::torrentCategoryChanged, this, &CategoryFilterModel::torrentCategoryChanged);
     connect(session, &Session::subcategoriesSupportChanged, this, &CategoryFilterModel::subcategoriesSupportChanged);
+    connect(session, &Session::finishedStartingUp, this,
+            [this, session]()
```

Comment:
I would move it on the previous line in all similar cases of this PR.

Reply:
Do you mean the lambda? Have it as a one liner?

Reply:
On 2nd thought, you probably mean only the capture clause and parameter list.

Reply:
>On 2nd thought, you probably mean only the capture clause and parameter list.

Yes.